src/test/java/com/ryanharter/auto/value/parcel/AutoValueParcelExtensionTest.java

```diff
@@ -159,12 +159,18 @@
         "import java.lang.String;\n" +
         "\n" +
         "final class AutoValue_Test extends $AutoValue_Test {\n" +
-        "  private static final ClassLoader CL = AutoValue_Test.class.getClassLoader();\n" +
+        "  static final ClassLoader CL = AutoValue_Test.class.getClassLoader();\n" +
         "\n" +
         "  public static final Parcelable.Creator<AutoValue_Test> CREATOR = new android.os.Parcelable.Creator<AutoValue_Test>() {\n" +
+        "\n" +
         "    @java.lang.Override\n" +
         "    public AutoValue_Test createFromParcel(android.os.Parcel in) {\n" +
-        "      return new AutoValue_Test(in);\n" +
+        "      return new AutoValue_Test(\n" +
+        "          in.readInt(),\n" +
+        "          in.readInt() == 0 ? (java.lang.Double) in.readSerializable() : null,\n" +
```

Comment:
Fully-qualified type names are needed inside constructor calls here (and other tests) because of a bug: https://github.com/google/compile-testing/issues/82